### ISSUE TITLE
500 trying to add bin manually

### ISSUE THREAD

**Issue by joefutrelle** [LouisK130/Manual-Classify#60]
For one of our time series we're getting the problem where when we try to add a bin manually, internally the bin ID is represented with its full URL, and so crashes the caching system.

We get this error on the backend. I suspect the issue is in the Javascript code that constructs the form parameters, still digging into that to see if I can figure out where the problem is being introduced.

Adding bins by time range works fine, and adding bins manually on other time series works fine.

```
[Thu Sep 13 17:00:31.432017 2018] [wsgi:error] [pid 19889]   File "/var/www/classify/classify/utils.py", line 96, in getTargets
[Thu Sep 13 17:00:31.432018 2018] [wsgi:error] [pid 19889]     new_targets = parseBinToTargets(bin, timeseries)
[Thu Sep 13 17:00:31.432020 2018] [wsgi:error] [pid 19889]   File "/var/www/classify/classify/utils.py", line 62, in parseBinToTargets
[Thu Sep 13 17:00:31.432021 2018] [wsgi:error] [pid 19889]     f = open(TARGETS_CACHE_PATH + '/' + bin + '_temp', 'w')
[Thu Sep 13 17:00:31.432024 2018] [wsgi:error] [pid 19889] FileNotFoundError: [Errno 2] No such file or directory: '/var/www/classify/classify/cache/targets/http://ifcb-data.whoi.edu/Attune_size_calibration_July2018/D20180912T193143_IFCB109_temp'
```

**Comment by joefutrelle** [comment]
A symptom is that in the "Select Bin(s)" box, after adding the bin, the full URL shows -- then adding the bin crashes. For other time series just the bin ID is shown.

**Comment by LouisK130** [comment]
I suspect this is another issue with the transition to HTTPS. I remembered to include a workaround for the search-by-date feature, but not manual validation apparently. I've included that in https://github.com/LouisK130/Manual-Classify/commit/eea80e5bfd5e6feb68be09e79e1e7675ffec1830.

That said, I'm not entirely sure why this timeseries would be the lone victim if that were the case. Somewhat suspicious. Let me know if this doesn't resolve the problem.
